src/drive/web/modules/drive/RenameInput.jsx

```diff
@@ -37,8 +50,12 @@ export const RenameInput = ({
       onSubmit={async newValue => {
         const newName = withoutExtension ? newValue + extension : newValue
         try {
-          await updateFileNameQuery(client, file, newName)
-          if (refreshFolderContent) refreshFolderContent()
+          if(isSanitized(newValue)) {
+            await updateFileNameQuery(client, file, newName)
+            if (refreshFolderContent) refreshFolderContent()
+          } else {
+            Alerter.error('alert.file_name_not_sanitized', { fileName: newName })
+          }
         } catch {
```

Comment:
```suggestion
        } catch (error) {
          if (error.message.includes('Invalid filename: empty or contains an illegal character')) { 
             Alerter.error('alert.file_name_not_sanitized', { fileName: newName })
          } else {
```

Reply:
in order to make frontend more resistant if backend updates the illegal characters list

Reply:
This suggestion would stop working the day we decide to change the error message on stack side. But I suppose the error code won't be enough to guess the error.

What is the risk of having this error message change in the future @nono ?

Reply:
This error message `The element %{fileName} contains invalid characters, please choose a new name.` is not helping the user to choose a new one. We should point out the forbidden character he used in the error's message in order to help the correction.

Reply:
> What are the chances of having this error message changed in the future @nono ?

Very low, we don't change error messages, except if we have a very good reason to do that.

Reply:
the check has been moved into cozy-client in this MR: https://github.com/cozy/cozy-client/pull/1085

I resolve this thread that is outdated now